src/styles/index.css

```diff
@@ -7,11 +7,32 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.app {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto 0 auto;
+  width: 600px;
+  height: 500px;
 }
 
-.text-center {
+.display {
+  background-color: grey;
+  color: #fff;
+  direction: rtl;
+  flex-direction: row;
+  font-size: 50px;
+  font-weight: bold;
+  height: 30px;
+  padding: 8px;
+}
+
+.btn-group {
+  height: 60px;
+}
```

Comment:
Requirement talks of using `100px` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png)]

Reply:
Oh, Thank you @shlock

Reply:
ooooh sorry about that